applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -895,8 +889,9 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
             for (IndexType point_number = 0; point_number < number_of_integration_points; ++point_number) {
                 // Compute element kinematics B, F, DN_DX ...
                 CalculateKinematicVariables(this_kinematic_variables, point_number, this->GetIntegrationMethod());
-
-                // Compute material reponse
+                // Compute constitutive law variables
+                SetConstitutiveVariables(this_kinematic_variables, this_constitutive_variables, Values, point_number, integration_points);
```

Comment:
?? what is this

Reply:
now i've added a bool to the `CalculateConstitutiveVariables` in order to know if it's rotated. Now they are back as it was